catroid/src/org/catrobat/catroid/physics/shapebuilder/PhysicsShapeBuilderStrategyFastHull.java

```diff
@@ -35,26 +35,26 @@
 
 public final class PhysicsShapeBuilderStrategyFastHull implements PhysicsShapeBuilderStrategy {
 
-	private final Stack<Vector2> convexHull = new Stack<Vector2>();
+	public static final int MINIMUM_PIXEL_ALPHA_VALUE = 1;
 
 	@Override
 	public Shape[] build(Pixmap pixmap, float scale) {
-
+		Stack<Vector2> convexHull = new Stack<Vector2>();
```

Comment:
A few lines done with it. Declare variables as close as possible to the first use.